game-core/src/main/java/games/strategy/engine/data/GameData.java

```diff
@@ -102,7 +102,6 @@
   private transient ResourceLoader resourceLoader;
   private IGameLoader loader;
   private History gameHistory = new History(this);
-  private transient volatile boolean testLockIsHeld = false;
```

Comment:
Use of this field was removed in #3715.  Because it is `transient`, removing it should not affect save game compatibility.